projects/storefrontlib/src/translations/en/my-account.en.ts

```diff
@@ -5,6 +5,10 @@ export const myAccount = {
     status: 'Status',
     shippedOn: 'Shipped on',
     inProcess: 'In process...',
+    READY_FOR_PICKUP: 'Ready for pickup',
```

Comment:
@mszczepaniak @Platonn what do you think about have orderStatus object for that?

Reply:
I am comfortable with both these solutions, so do not have any real preference here, how about you @Platonn ?

Reply:
I'd rather use special `context` translation parameter which we support https://www.i18next.com/translation-function/context . We should have it documented soon.
html:
```
{{ 'orderDetails.deliveryStatus' | cxTranslate : { context: consignment.status } }}
```

resources:
```
deliveryStatus_READY_FOR_PICKUP: 'Ready for pickup'
...
```

Then is no need to add method `addLocalizationPrefix` and store prefix in local variable of the component.

Reply:
Ok, thank you for the information. Refactored